ci/jenkins/jobs/projects.yaml

```diff
@@ -954,3 +954,86 @@
              - timeout:
                  timeout: 600
                  type: absolute
+      - '{name}-{test_name}-matrix-compatibility-test':
+          test_name: upgrade
+          node: 'antrea-test-node'
+          description: 'This is the upgrade matrix compatibility test for Antrea. In this test, more than half of all Antrea agents and Antrea controller are old-versioned.'
```

Comment:
does that make sense? I believe we assume that the controller is updated first
https://github.com/vmware-tanzu/antrea/blob/master/docs/versioning.md#antrea-upgrade-and-supported-version-skew

I know that some changes we did in the past would fail without this assumption. For example, when updating the controlplane API version, a new controller would support both API versions but each agent would only support a single API version (old version for old agents and new version for new agents).

@tnqn

Reply:
After discussion with @edwardbadboy , I updated the code and now the Antrea controller is using the latest image in this job.